.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,29 +1,12 @@
 # Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
 blank_issues_enabled: false  # default: true
 contact_links:
-- name: 🔐 Security bug report 🔥
```

Comment:
Things like this mustn't be a part of the PR.

Reply:
Sorry, I  wrongly operated git command and included someone else's commit to my PR.